cypress/integration/QAM/TC01_basics.spec.js

```diff
@@ -0,0 +1,8 @@
+describe('Login', function(){
```

Comment:
Tu i wszędzie gdzie masz `function()` - rozważ używanie arrow function `() =>`. Działa tak samo (z pominięciem kontekstu `this`, ale to Cię nie interesuje), a jest krótsze.